core/platform/search/elastic_search_services.py

```diff
@@ -26,13 +26,16 @@
 
 import elasticsearch
 
+# This timeout is selected because 30 seconds results in reloading a
+# exploration and exp_services.load_demo() failing with a ReadTimeoutError
+# if the 30 seconds timeout isn't used.
```

Comment:
This sentence is run-on--can you revise it to be a bit clearer & easier to read?